Index: src/App.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import './App.css';\nimport {useSelector} from \"react-redux\";\nimport DrumPads from \"./components/DrumPads\";\nimport Controls from \"./components/Controls\";\nfunction App() {\n\n\n  return (\n    <section className=\"App\" id=\"drum-machine\">\n        <div id={\"display\"}>\n            <DrumPads/>\n            <Controls/>\n        </div>\n\n    </section>\n  );\n}\n\nexport default App;\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/App.js b/src/App.js
--- a/src/App.js	(revision 56ebd5de218039cf54c3448dcb73bcade898b84d)
+++ b/src/App.js	(date 1622199164270)
@@ -1,7 +1,7 @@
 import './App.css';
-import {useSelector} from "react-redux";
 import DrumPads from "./components/DrumPads";
 import Controls from "./components/Controls";
+
 function App() {
 
 
Index: src/setupTests.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>// jest-dom adds custom jest matchers for asserting on DOM nodes.\n// allows you to do things like:\n// expect(element).toHaveTextContent(/react/i)\n// learn more: https://github.com/testing-library/jest-dom\nimport '@testing-library/jest-dom';\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/setupTests.js b/src/setupTests.js
--- a/src/setupTests.js	(revision 56ebd5de218039cf54c3448dcb73bcade898b84d)
+++ b/src/setupTests.js	(date 1622199164354)
@@ -2,4 +2,4 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom';
+import '@testing-library/jest-dom';
\ No newline at end of file
Index: src/App.test.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import { render, screen } from '@testing-library/react';\nimport App from './App';\n\ntest('renders learn react link', () => {\n  render(<App />);\n  const linkElement = screen.getByText(/learn react/i);\n  expect(linkElement).toBeInTheDocument();\n});\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/App.test.js b/src/App.test.js
--- a/src/App.test.js	(revision 56ebd5de218039cf54c3448dcb73bcade898b84d)
+++ b/src/App.test.js	(date 1622199164302)
@@ -1,4 +1,4 @@
-import { render, screen } from '@testing-library/react';
+import {render, screen} from '@testing-library/react';
 import App from './App';
 
 test('renders learn react link', () => {
